deploy/oasis-v2/skillopt/skillopt_runner.py

```
@@ -0,0 +1,425 @@
+            interesting = {key: payload[key] for key in payload if key in {"best_score", "current_score", "selection_hard", "selection_soft", "test_hard", "test_soft", "accepted", "gate"}}
+            if interesting:
+                summaries.append({"file": str(path.relative_to(output_dir)), "values": interesting})
+    return {"score_artifacts": summaries[:30]}
+
+
+def run_learning(trigger: str) -> dict[str, Any]:
+    if not ENABLED:
+        raise ValueError("SkillOpt est désactivé par OASIS_SKILLOPT_ENABLED.")
+    api_key = os.environ.get("OPENROUTER_API_KEY")
+    if not api_key:
+        raise ValueError("OPENROUTER_API_KEY est requis pour l’apprentissage SkillOpt.")
```

Ruff (0.16.2)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)